cylc/flow/data_messages_pb2.py

```diff
@@ -1,12 +1,22 @@
 # -*- coding: utf-8 -*-
 # Generated by the protocol buffer compiler.  DO NOT EDIT!
+# NO CHECKED-IN PROTOBUF GENCODE
 # source: data_messages.proto
-# Protobuf Python Version: 4.25.3
+# Protobuf Python Version: 5.28.0
```

Comment:
Not regenerated since upping to 6.32?

Reply:
If the support bracket is as stated, then this file should not need to be recompiled for this change.

And if we were to recompile this file, it should be done using the oldest supported version:

> gencode should never be paired with a runtime that predates the protoc and plugin that was used to generate those bindings.
>
> -- https://protobuf.dev/support/cross-version-runtime-guarantee/#backwards